mangaki/mangaki/migrations/0088_auto_20171225_1534.py

```diff
@@ -66,10 +66,11 @@ def copy_source_field_to_reference(apps, schema_editor):
         try:
             source = infer_source(work.source.lower())
             identifier = infer_identifier(work.source, source)
-            ref = Reference.objects.get_or_create(work=work,
-                            source=source,
-                            identifier=identifier,
-                            url=work.source)
+            if identifier is not None:
+                ref = Reference.objects.get_or_create(work=work,
```

Comment:
__fake__.MultipleObjectsReturned: get() returned more than one Reference -- it returned 2!

Reply:
Je vais revérifier, mais pas normal.